docs/install/osx_setup.md

```diff
@@ -145,12 +155,16 @@ We have installed MXNet core library. Next, we will install MXNet interface pack
 To install the MXNet Python binding navigate to the root of the MXNet folder then run the following:
 
 ```bash
-$ cd python
-$ pip install -e .
+virtualenv -p`which python3` mxnet_py3
```

Comment:
While I generally agree that virtualenv is a good practice, it adds another dependency.

These guides are for the very basic user to get something running. I think this PR overcomplicates a lot of things. Adding the cmake compilation? Fine! Everything else? Please leave it as it is.